vault/core.go

```diff
@@ -2069,6 +2069,7 @@ func stopReplicationImpl(c *Core) error {
 func (c *Core) emitMetrics(stopCh chan struct{}) {
 	emitTimer := time.Tick(time.Second)
 	writeTimer := time.Tick(c.counters.syncInterval)
+	identityCountTimer := time.Tick(time.Minute)
```

Comment:
I had been looking at one measurement per 10 minutes. I'm having an active discussion with @briankassouf about how often we can reasonably call a storage List() operation that may have a large number of entries.

I think the UI does *not* auto-refresh, so I we don't really have a measure of "how often is this count being performed today" as a baseline.

Reply:
Luckily this count is done all in-memory and does not have to hit storage. I also don't think it will lock anything (though we should confirm) since we can use a read txn on memdb. It may not be an issue to do this every minute, but doing it every 10 minutes also seems fine.

Reply:
Oh, sorry, countEntities() uses MemDb, it's countActiveTokens() that does a storage list.

Reply:
10 minutes (or more, even) seems fine to me. It is a somewhat expensive operation, `time vault read -format=json sys/internal/counters/entities` takes ~1.9s for our current 5.2 million entities. I'll update to 10 mins.

Reply:
There's no lock in countEntities(), it's just a normal read transaction.  10 minute seems appropriate based on Dustin's measurement (thanks for sharing that!)